network-api/networkapi/donate_banner/models.py

```diff
@@ -64,15 +95,19 @@ class DonateBanner(TranslatableMixin, PreviewableMixin, models.Model):
         FieldPanel("subtitle"),
         FieldPanel("cta_button_text"),
         FieldPanel("cta_link"),
-        FieldPanel("background_image"),
+        FieldPanel("foreground_image"),
+        FieldPanel("bg_image"),
+        FieldPanel("bg_color"),
```

Comment:
```suggestion
        FieldPanel("foreground_image"),
        FieldPanel("background_image"),
        FieldPanel("background_color"),
```
to make naming convention consistent